json-guides/circuit-breaker.json

```diff
@@ -74,7 +74,7 @@
             "name": "AddLibertyMPFaultTolerance",
             "title": "Enabling MicroProfile Fault Tolerance",
             "description": [
-                "Begin by enabling the MicroProfile Fault Tolerance 1.0 feature in your <code>server.xml</code> file.  This feature is required and provides an environment to support resilient microservices through patterns that include circuit breakers and fallbacks."
+                "Microprofile fault tolerance is a way for microservices to handle the unavailability of a service by using different policies to guide the execution and result of some logic.  The MicroProfile Fault Tolerance 1.0 feature provides an environment to support resilient microservices through patterns that include circuit breakers and fallbacks.  Enable the MicroProfile Fault Tolerance 1.0 feature in the server.xml of the Liberty server where the Check Balance microservice runs."
```

Comment:
Should `server.xml` still have the `<code>` styling?

Reply:
Definitely should! ...Thanks for noticing that ....fixing